hive-metastore/src/main/java/org/apache/iceberg/hive/HiveCatalog.java

```diff
@@ -539,12 +541,26 @@ Database convertToDatabase(Namespace namespace, Map<String, String> meta) {
             database.setDescription(value);
           } else if (key.equals("location")) {
             database.setLocationUri(value);
+          } else if (key.equals(TableProperties.HMS_DB_OWNER)) {
+            database.setOwnerName(value);
+          } else if (key.equals(TableProperties.HMS_DB_OWNER_TYPE)) {
+            database.setOwnerType(PrincipalType.valueOf(value));
           } else {
             if (value != null) {
               parameter.put(key, value);
             }
           }
         });
+
+    // default ownership on db level kept uniform with table level, see  HiveTableOperations#
```

Comment:
I think the code comment is unnecessary (like, example, why not comment the other method and say its kept uniform with this one..), I would say remove comment entirely.

Reply:
ack

Reply:
comment removed in the latest commit